packages/reality-data/public/locales/en/IModelSelect.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
I dont think you meant to add this file. This should probably be replaced with src/public/locales/en/RealityData.json